index.md

```diff
@@ -342,7 +345,7 @@ more about the payment instrument specific problem messages below:
   [iso-3166]: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
   [uuid]: https://en.wikipedia.org/wiki/Universally_unique_identifier
   [external-integration]: https://api.externalintegration.payex.com
-  [production]: https://api.payex.com/
+  [production]: https://api.externalintegration.payex.com/
```

Comment:
This link is a lie.
When used the user expect this to point to prod, as supported by the text around it, but here it is edited to be external integration.

Reply:
Good catch. It was an unintended search & destroy. Will fix.